playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/PaymentVerifier.java

```diff
@@ -31,6 +31,9 @@
     // TODO: Should this be an instance class (eg PaymentStrategy)?
     // It would allow developers to override verification behaviour more easily.
 
+    private static final String ARC_FEATURE = "org.chromium.arc";
```

Comment:
This is also defined on the main library: https://github.com/GoogleChrome/android-browser-helper/blob/master/androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java#L46

Since billing depends on that library, I'm wondering if we should move the variable to an appropriate and make it accessible by the billing library so we can re-use.

Reply:
I've create the support class. Please take a look again!